src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/DefaultLanguageServerFeatureOptions.cs

```diff
@@ -44,4 +44,6 @@ public override bool ReturnCodeActionAndRenamePathsWithPrefixedSlash
     public override bool UseRazorCohostServer => false;
 
     public override bool DisableRazorLanguageServer => false;
+
+    public override bool EnableFuse => false;
```

Comment:
Happy if this is going to be renamed in a future commit before merging, but we probably should use a name that means something, not a code name. I think "SuppressDesignTime" is fine, and matches the compiler.

Reply:
I'll rename, but here's my 2c https://github.com/dotnet/razor/pull/9964#discussion_r1496743635 

I definitely don't care enough either way. So if as a reader `SuppressDesignTime` is better I'm all game

Reply:
I think I commented before you wrote yours. I think your reasoning is good, but I still don't think we should have "Fuse" in the name. What about just "AlwaysPerformRuntimeCodeGeneration" ?

Reply:
Totally up to you, but if you want to adjust the compiler side to match the terminology that's fine with me, just so we're consistent (also fine to not be, this won't be around for too long anyway (he said, confidently))